animations/implicit_widgets.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: page
+title: "Flutter implicit animations"
+permalink: /animations/implicit_widgets.html
+---
+
+* TOC Placeholder
+{:toc}
+
+
+If you’re new to Flutter or new to animation, consider starting with the built-in Flutter implicit widgets. Implicit animations are useful when you want to quickly add an animation without having to write a lot of animation code because the Flutter SDK implicit widgets already include the `AnimationController`, the ticker provider, the listener, and other animations elements.
```

Comment:
How about "Implicit animations are useful when you want to quickly add ~~an animation~~ a common motion effect to your app." I'd delete the rest of the sentence, for two reasons 1) it discourages the user to explore the other two implementation approaches, and 2) it brings up a number of concepts the user doesn't need to know at this point.